lib/lib/component.tsx

```diff
@@ -3,7 +3,7 @@ import { cn } from "@/lib/utils"
 import { forwardRef } from "react"
 import { useComponentXRay } from "./xray"
 
-export const componentTypes = ["layout", "info", "action"] as const
+export const componentTypes = ["layout", "info", "action", "form"] as const
```

Comment:
Awesome! X-Ray should eventually create a tiny interface that allows us to toggle and filter what we want to see, so this makes sense.